pandas_access/__init__.py

```diff
@@ -14,6 +16,19 @@
 
 DEF_RE = re.compile("\s*\[(\w+)\]\s*(.*)$")
 
+# Get executable directory
+env = os.environ.get('VIRTUAL_ENV', None)
+if env is None:
+    bin_dir = ''
+else:
+    if platform.system() == 'Windows':
+        bin_dir = os.path.join(env, 'Scripts')
+    else:
+        bin_dir = os.path.join(env, 'bin')
+
+mdb_export = os.path.join(bin_dir, 'mdb-export')
+mdb_schema = os.path.join(bin_dir, 'mdb-schema')
+mdb_tables = os.path.join(bin_dir, 'mdb-tables')
```

Comment:
Might be good to do an `if os.path.isfile` on each of these, and revert to the un-prefixed version if not.

Reply:
Now I'm only invoking the absolute path generation if its not already on the path.

Reply:
That's an improvement. It still might be good to check if there's an executable at the absolute path so that we can raise a more helpful error message if not, though.

Reply:
I guess the error would need to be raised in each function, though, not during module init.